components/camera/align/extrinsics.go

```diff
@@ -146,10 +146,15 @@ func newColorDepthExtrinsics(ctx context.Context, color, depth camera.Camera, at
 		debug:     attrs.Debug,
 		logger:    logger,
 	}
+	var cameraModel transform.PinholeCameraModel
+	cameraModel.PinholeCameraIntrinsics = attrs.CameraParameters
```

Comment:
Due to the reuse of this code, do we want to consider creating a function that will give `cameraModel` from a given attrs? (This might be what eric was already referring to)